src/configure.in

```diff
@@ -118,15 +118,34 @@ LIBUTIL=-lutil
 ])
 AC_SUBST(LIBUTIL)
 
-AC_CHECK_HEADER(libintl.h, [
-	AC_SEARCH_LIBS(dgettext, intl, [
-		AC_DEFINE(ENABLE_NLS, 1,
-			[Define if translation functions should be used.])])])
-
-AC_CHECK_PROG(MSGFMT,msgfmt,msgfmt)
+# Determine if NLS is desired and supported.
 po=
-if test x"$MSGFMT" != x; then
-	po=po
+AC_ARG_ENABLE([nls],
+AC_HELP_STRING([--disable-nls],
+               [Disable Native Language Support(NLS).]), explicit=yes,
```

Comment:
This file has a mix of tabs and space but the majority of the file is indented like this with spaces. All the AC_HELP_STRING that are multi-line are like this (except 2 with have one tab and then spaces).

Seems better to stick with the existing style unless there's a real compelling reason.